web/views/pages/index.ejs

```diff
@@ -94,7 +94,7 @@
         </div>
         <svg class="map sea"></svg>
         <canvas class="map wind"></canvas>
-        <canvas class="map solar"></canvas>
+        <canvas class="map solar" id="solar-map"></canvas>
```

Comment:
Is is possible to change the `stackBlurCanvasRGBA` function to take a css selector instead? That would be cleaner.
Else we should change the `draw` function to take a `canvasId` instead of a `canvasSelector`, and then we can pass the id all the way instead of relying on a class selector.